sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -549,7 +549,7 @@ object functions {
    * The function by default returns the last values it sees. It will return the last non-null
    * value it sees when ignoreNulls is set to true. If all values are null, then null is returned.
    *
-   * @note The function is non-deterministic because its results depends on order of rows which
+   * @note The function is non-deterministic because its results depends on the order of rows which
```

Comment:
Total nit, but you use "on the order or rows" here but "on order of the rows" above. The most correct would be 'the order of the rows' but honestly doesn't matter much. Feel free to standardize on anything

Reply:
Oops